wsmaster/codenvy-hosted-platform-api-impl/src/test/java/com/codenvy/api/workspace/TestObjects.java

```diff
@@ -64,7 +64,8 @@ public static WorkspaceImpl createWorkspace(String owner, String devMachineRam,
         Map<String, ExtendedMachineImpl> machines = new HashMap<>();
         machines.put("dev-machine", new ExtendedMachineImpl(singletonList("org.eclipse.che.ws-agent"),
                                                             emptyMap(),
-                                                            new HashMap<>(singletonMap("memoryLimitBytes", Long.toString(Size.parseSize(devMachineRam))))));
+                                                            new HashMap<>(singletonMap("memoryLimitBytes",
```

Comment:
Does the map have to be mutable at this point?
If so, than `ExtendedMachineImpl` constructor should be responsible for that, if not
you can simply pass the instance of `singletonMap(...)`